### PR TITLE
Astrodyne inlet rev b

### PR DESCRIPTION
**Pull request by wrex**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

Update to the README for the astrodyne 082S/082SM power inlet mod. The tabs that I thought were just to prevent non-medical-certified power cords from being inserted are actually present on **both** the general-purpose and medical version, and are simply there to hold the power cord more securely. Basically, I just needed to push harder!

#### How was it tested? 

Ordered both a 082S and 082SM power inlet, and verified I was able to insert normal C13 power cords once I'd "persuaded" the tabs a bit. The first insertion is pretty difficult, but after multiple insertions it frees up a bit.

#### Any background context you want to provide?

Rex is an idiot. Sorry for the fire drill!

### CODE REVIEW

**Comment by wrex** [comment]
Geez this was embarassing. The tabs on the sides seem unnecessary to me, but they are definitely present in both medical and general-purpose models, and they do bend out of the way (with sufficient effort).

File: printer_mods/wrez/astrodyne-inlet/README.md [hunk 1]
```diff
@@ -1,28 +1,51 @@
...
-(which has been on back-order at Digikey for quite a while at the time of
-this writing) with an
-[Astrodyne
-082S/082SM](https://www.mouser.com/ProductDetail/astrodyne-tdi/082sm0150000/?qs=eP2BKZSCXI4KQZVxn1eqpw%3D%3D).
+(which has been on back-order at Digikey for quite a while at the time of this
+writing) with an [Astrodyne/TDI
+082S/082SM](https://www.mouser.com/ProductDetail/Astrodyne-TDI/082S0150000?qs=eP2BKZSCXI7jG836kH3rhw%3D%3D).
+
+In addition to the power inlet you will also need to acquire some inexpensive 5
+mm diameter X 20 mm long 250V fast-acting fuses (either 15A or 10A fuses are
```
Comment (KiloQubit) [inline printer_mods/wrez/astrodyne-inlet/README.md:11]: Please don't use 15A or 20A fuses in a V0.  The sourcing guide shows a 2.5A fuse, and also has a link to one at Digikey.  The 18 gauge wire used for mains in the manual can't even take 20A, and of course the 150W PSU will never draw that much under normal conditions.  I would either remove this whole paragraph, or just change it to say that fuses will need to be purchased and there's a link in the sourcing guide.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the update!  I have a change to the README below.

File: printer_mods/wrez/astrodyne-inlet/README.md (hunk 1 above)
Comment (wrex) [reply printer_mods/wrez/astrodyne-inlet/README.md]: Yup. Momentary insanity. Changes pushed.

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!

**Comment by KiloQubit** [comment]
It looks like you have a merge conflict in your README, please resolve it.

**Pull request merged**